packages/inter-protocol/src/proposals/deploy-price-feeds.js

```diff
@@ -139,6 +141,15 @@ const startPriceAggregatorInstance = async (
     // @ts-expect-error GovernableStartFn vs. fluxAggregatorContract.js start
     installation,
   });
+  const retiringInstance = await consumeInstance[AGORIC_INSTANCE_NAME];
+  const retiredContractInstances = await retiredContractInstancesP;
+  retiredContractInstances.init(
+    // XXX tail of label needs to vary with upgrade. BundleId would be different
+    // from previous, but is not necessarily unique.
+    `priceFeed-${AGORIC_INSTANCE_NAME}-u18`,
```

Comment:
Yeah the `-u18` suffix is probably the best we can do, short of using a list and appending to it (which would make future proposals that need to reference a specific one harder to write and review). At least if someone re-uses this proposal and forgets to update the suffix, that `.init()` will throw and we should catch it during testing.

Reply:
boardID works better and can be automated.